source/cloud/cloud-administration/site-configuration.rst

```diff
@@ -32,27 +32,27 @@ Mattermost in-product notices keep users and administrators informed of the newe
 Administrator Notices
 ^^^^^^^^^^^^^^^^^^^^^
 
-Administrator notices are used to inform System Admins when a new server version is available, or if a server upgrade is recommended due to ending support life cycles. System Admins may also receive notices about recommended server configuration options to optimize the user experience of their deployment.
+Administrator notices are used to inform System Admins of important updates or recommended configuration options to optimize the user experience of their deployment.
 
-Administrator notices can be disabled in the **System Console > Notices** page.
+Administrator notices can be disabled in the **System Console > Site Configuration > Notices** page.
```

Comment:
@adamjclarkson given that we would be the ones sending Administrator Notices, if any, should they be able to disable them in Cloud?

Reply:
@wiersgallak thoughts on removing the ability for admins to disable Notices in cloud servers?

Reply:
^ @adamjclarkson

Reply:
Agree we shouldn't allow admins to disable notices.

Reply:
^ @marianunez - I'm going to take the details about how to disable admin notices out of the Cloud Admin Guide documentation as per PM feedback. Can we also remove this functionality from the Cloud System Console?

Reply:
Thanks @cwarnermm! Created a ticket https://mattermost.atlassian.net/browse/MM-33500